sunpy/map/sources/hinode.py

```diff
@@ -98,6 +98,20 @@ def processing_level(self):
             return
         return int(lvl)
 
+    @property
+    def unit(self):
+        # XRT data values are in DN and are converted into DN/s if the data has been normalized.
+        # A tag starting with "XRT_RENORMALIZE" is added to the HISTORY tag in that case.
+        # Ref. Section 2.11 of the XRT Analysis Guide.
+        unit = self.meta.get('BUNIT')
+        if not unit:
+            unit = u.astrophys.dn
```

Comment:
I forgot now, I will need to check with the others, but we don't use DN but counts as a result that we can't save out Maps with DN units.